CHANGELOG-canary.md

```diff
@@ -0,0 +1,13 @@
+## 18.3.0-canary-546178f91-20231005 (October 5, 2023)
```

Comment:
Since we publish these every day, I would probably deemphasize the specific version in the title and use the date as the header instead

Reply:
Fixed